src/odemis/driver/orsay.py

```diff
@@ -2357,7 +2357,7 @@ def on_connect(self):
         self._objectiveVoltageConnector = OrsayParameterConnector(self.objectiveVoltage, self._hvps.ObjectiveVoltage,
                                                                   minpar=self._hvps.ObjectiveVoltage_Minvalue,
                                                                   maxpar=self._hvps.ObjectiveVoltage_Maxvalue)
-        self._beamShiftConnector = OrsayParameterConnector(self.beamShift, [self._ionColumn.ObjectiveShiftX,
+        self._shiftConnector = OrsayParameterConnector(self.shift, [self._ionColumn.ObjectiveShiftX,
                                                                             self._ionColumn.ObjectiveShiftY],
```

Comment:
now that you've shortened the name I think it's a bit nicer to re-align this line with the one above